### DISCUSSION TITLE
求助！请问手机APP无法方位EX 但是网页可以访问！ 应该如何操作？！

### DISCUSSION THREAD

**Discussion by Andreariel2022** [xiaojieonly/Ehviewer_CN_SXJ#1707] [Q&A]
手机APP重新登录了，有人说是igneousID的问题 也都刷新了，还有的说是最近EH有了机器人 需要人机认证重新登陆即可，也试了依旧没用！求大佬给点办法！！！！

**Comment by Technistretron** [comment]
我自己遇到的问题，可能有启发，我软路由建立了针对exhentai的dns分流，但是用app永远进不去里站，但是网页可以，最后通过流量分析发现是因为ehviewerapp里面自带了host，锁定了exhentai的解析地址，在高级里面把内置的host关掉就好了

**Reply by Technistretron** [reply]
哦，好像还要把域名前置关闭

**Reply by imleafs** [reply]
还是不得行啊

**Reply by LiniJar** [reply]
谢谢你，已经解决了这个问题，但是现在APP只能用梯子访问里站了😇
